Gopkg.lock

```diff
@@ -1,6 +1,14 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
 
 
+[[projects]]
+  branch = "master"
+  digest = "1:e95c8ee2ac122e3b55a7bef845c4c65bbd240685da1dbb48aee2d6854e16c758"
+  name = "github.com/alecthomas/log4go"
```

Comment:
are we explicitly adding log4go package  or added by mattermost server ?

Reply:
Will check on that and also push the pending mattermost-server dependencies in vendor.

Reply:
Checked on the dependencies. All the added dependencies are required for Mattermost.